### PR TITLE
Issue #91 - Restructure

### PR DESCRIPTION
**Pull request by CrayCrayDecay**
Restructured the application to use the "dist" folder rather than the "docs" folder. I also updated the readme to reflect this change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
   "license": "MIT",
   "dependencies": {
     "d3": "^5.7.0",
+    "gulp-sass": "^4.0.2",
```
Comment (BurkusCat) [inline package.json]: Great work on simplifying the setup process!

**Comment by BurkusCat** [comment]
I'm still able to see the 'docs' directory with a LICENSE file in it. Would you be able to remove that please?

**Comment by BurkusCat** [comment]
By default, the 'dist' directory has a few files in it that are in the repo. Should these be removed?

**Comment by BurkusCat** [comment]
@CrayCrayDecay this is fantastic work. This will make a big difference in ease of setup, contributing, testing etc. Thank you!

I need to do some work on the AWS side to install node modules, run gulp tasks and sync the site from the dist folder instead of docs. Previously I was just syncing the docs folder. So it may be a couple of days before I am able to merge this in.

**Comment by CrayCrayDecay** [comment]
1. I removed the license file from the docs directory.
2. You should be able to remove the docs folder from the repo completely after testing.

**Pull request merged**